src/DssSpell.t.sol

```diff
@@ -1371,337 +1377,80 @@ contract DssSpellTest is DssSpellTestBase {
 
     // SPELL-SPECIFIC TESTS GO BELOW
 
-    function testRewardsDistUsdsSkyUpdatedVestIdAndDistribute() public {
-        address REWARDS_DIST_USDS_SKY = addr.addr("REWARDS_DIST_USDS_SKY");
-        address REWARDS_USDS_SKY = addr.addr("REWARDS_USDS_SKY");
-
-        uint256 vestId = VestedRewardsDistributionLike(REWARDS_DIST_USDS_SKY).vestId();
-        assertEq(vestId, 5, "TestError/rewards-dist-usds-sky-invalid-vest-id-before");
-
-        uint256 unpaidAmount = vestSky.unpaid(5);
-        assertTrue(unpaidAmount > 0, "TestError/rewards-dist-usds-sky-unpaid-zero-early");
+    function testNovaOperatorOffboarding() public {
+        address roles     = addr.addr("ALLOCATOR_ROLES");
+        address buffer    = addr.addr("ALLOCATOR_NOVA_A_BUFFER");
+        address vault     = addr.addr("ALLOCATOR_NOVA_A_VAULT");
+        address operator  = wallets.addr("NOVA_OPERATOR");
+        address novaProxy = 0x355CD90Ecb1b409Fdf8b64c4473C3B858dA2c310;
 
-        _vote(address(spell));
-        _scheduleWaitAndCast(address(spell));
-        assertTrue(spell.done(), "TestError/spell-not-done");
 
-        vestId = VestedRewardsDistributionLike(REWARDS_DIST_USDS_SKY).vestId();
-        assertEq(vestId, 6, "TestError/rewards-dist-usds-sky-invalid-vest-id-after");
+        // Sanity check values before spell
+        assertEq(usds.allowance(buffer, operator), type(uint256).max, "invalid-allowance-before");
+        assertTrue(AllocatorRolesLike(roles).hasUserRole("ALLOCATOR-NOVA-A", operator, 0));
+        assertTrue(AllocatorRolesLike(roles).hasActionRole("ALLOCATOR-NOVA-A", vault, AllocatorVaultLike.draw.selector, 0));
+        assertTrue(AllocatorRolesLike(roles).hasActionRole("ALLOCATOR-NOVA-A", vault, AllocatorVaultLike.wipe.selector, 0));
```

Comment:
The values checked here are the same as the values checked on L1409 and L1410, after the spell is cast.
This means that the state is not being altered.
Why do we have instructions to perform this "clean up" if nothing needs to be cleaned up?